src/App.tsx

```diff
@@ -10,33 +11,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState('tab-1');
+  const handleTabSelection = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    event.preventDefault();
+    if (event.currentTarget) {
+      setSelectedTab(event.currentTarget.id);
+    }
```

Comment:
don't pass event in `handleTabSelection` function, tab id will be enough and you can remove logic with event (preventDefault, currentTarget)